cmd/jaeger/internal/extension/jaegerquery/server.go

```diff
@@ -79,18 +80,20 @@ func (s *server) Start(_ context.Context, host component.Host) error {
 	if err != nil {
 		return fmt.Errorf("could not initialize a tracer: %w", err)
 	}
-
+	telset := telemetery.Setting{
+		Logger:       s.logger,
+		Tracer:       s.jtracer,
+		ReportStatus: telemetery.HCAdapter(healthcheck.New()),
```

Comment:
In v2 there is no reason to use HC, just use the status reporter already provided by otelcol